### ISSUE TITLE
Feature Request: Option to change FF behaviour between Hold and Toggle

### ISSUE THREAD

**Issue by lukehook** [budude2/openfpga-GBC#71]
Apologies if this already exists but I couldn't find the option or any mention of it in the existing issues or even on the reddit.

Would be great if we could toggle (or perhaps even just change the default behaviour?!?) of the FF button between 'Hold' and 'Toggle'. I know we have the option to remap currently but I can't see anyway of being able to maintain the FF functionality other than manually holding the button down.

For some games, e.g. Pokemon. You may want to be in FF mode for a substantial period of time, maybe even for the whole play session. So holding down the button can become tiresome.

**TL:DR;**

- **Introduce a way of adjusting the FF behaviour between Toggle and Hold**

**Comment by lukehook** [comment]
Closing dude to my own stupidity as the option already exists
